io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl

```diff
@@ -0,0 +1,213 @@
+##########################################################################################
+# Copyright (c) 2023,2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e. V.
+# Copyright (c) 2023,2024 German Edge Cloud GmbH & Co. KG
+# Copyright (c) 2023,2024 Siemens AG
+# Copyright (c) 2023,2024 Contributors to the Eclipse Foundation
+
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+##########################################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.material_flow_scenario_request:2.0.0#> .
+@prefix ext-result: <urn:samm:io.catenax.material_flow_simulation_result:3.0.0#> .
+
+:MaterialFlowScenarioRequestAspect a samm:Aspect ;
+   samm:preferredName "Material Flow Scenario Request Aspect"@en ;
+   samm:description "Aspect model describing the Material Flow Scenario Request"@en ;
+   samm:properties ( :materialFlowScenarioRequest ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:materialFlowScenarioRequest a samm:Property ;
+   samm:preferredName "materialFlowScenarioRequest"@en ;
+   samm:description "Contains standardized attributes for the content of the Material Flow Scenario Request"@en ;
+   samm:characteristic :MaterialFlowScenarioRequestCharacteristic .
+
+:MaterialFlowScenarioRequestCharacteristic a samm:Characteristic ;
+   samm:preferredName "Material Flow Scenario Request Characteristic"@en ;
+   samm:description "Characteristic describing the aspect Material Flow Scenario Request"@en ;
+   samm:dataType :MaterialFlowScenarioRequest .
+
+:MaterialFlowScenarioRequest a samm:Entity ;
+   samm:preferredName "Material Flow Scenario Request"@en ;
+   samm:description "The Material Flow Scenario Request contains standardized attributes for exchanging scenario requests and the resulting simulation results between two OSIM Manager Instances"@en ;
+   samm:properties ( :scenarioHeader [ samm:property :scenarioParameter; samm:optional true ] :scenarioSimResults ) .
+
+:scenarioHeader a samm:Property ;
+   samm:preferredName "scenarioHeader"@en ;
```

Comment:
preferredName should be human readable and follow normal orthography (e.g., no camel case but normal word separation)
Please correct this in other places also in model.

Reply:
Updated preferred names